README.md

```diff
@@ -328,6 +328,7 @@ Table of Contents
   * [travis-ci.org](https://travis-ci.org/) — Free for public GitHub repositories
   * [cirrus-ci.org](https://cirrus-ci.org) - Free for public GitHub repositories
   * [drone](https://cloud.drone.io/) - Drone Cloud enables developers to run Continuous Delivery pipelines across multiple architectures - including x86 and Arm (both 32 bit and 64 bit) - all in one place
+  * [ainize.ai](https://www.ainize.ai/) - CD service availabe for Open Source. A project can utilize up to 2GB of memory and 1 vCPU (n1-standard-1, 3.75GB). Github integration
```

Comment:
this says it's a beta and has no details about the future packages availability, as per the top of README I can only take things in Beta with certain constraints.

Also it needs to force the use of https.